src/style.css

```diff
@@ -1 +1,73 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+html {
+  font-weight: 300;
+}
+
+body {
+  margin: 0;
+}
+
+.form:not(:last-child) {
+  margin-bottom: 20px;
+  margin-top: 20px;
+}
+
+.searchbar__container {
+  position: relative;
+}
+
+.searchbar {
+  width: 100%;
+  background-image: url('./images/Search.svg');
+  background-repeat: no-repeat;
+  box-sizing: border-box;
+  font-family: Avenir, Arial, sans-serif;
+  font-weight: 400;
+  border-radius: 4px;
+  border: 1px solid rgb(225, 231, 237);
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  transition: all 0.15s ease-in-out;
+  outline: none;
+}
+
+.searchbar--big {
+  background-size: 19px;
+  background-position: 26px center;
+  height: 70px;
+  font-size: 16px;
+  line-height: 22px;
+  padding-left: 62px;
+}
+
+.searchbar--small {
+  background-size: 11px;
+  background-position: 13px center;
+  height: 42px;
+  padding-left: 34px;
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.searchbar--big:hover,
+.searchbar--small:hover {
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+}
+
+.searchbar--small:focus,
+.searchbar--big:focus {
+  border-color: rgb(225, 231, 237);
+  font-weight: 900;
```

Comment:
Avoid changing font-weight on focus as it can cause layout shifts. Consider using other styles like border or color changes to indicate focus.